src/main/java/common/Iterables.java

```diff
@@ -0,0 +1,10 @@
+package common;
+
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+public final class Iterables {
+    public static <T> Stream<T> stream(Iterable<T> i) {
+        return StreamSupport.stream(i.spliterator(), false);
+    }
+}
```

Comment:
What are the advantages of using this and why don't we use Stream.of(...) method ?